freqtrade/persistence/trade_model.py

```diff
@@ -680,24 +700,33 @@ def update_trade(self, order: Order) -> None:
             if self.is_open:
                 payment = "SELL" if self.is_short else "BUY"
                 logger.info(f'{order.order_type.upper()}_{payment} has been fulfilled for {self}.')
+
+            # TODO WIP but to rm if useless
+            # condition to avoid reset value when updating fees (new)
+            # if order.order_id in self.open_orders_ids:
+            #    self.open_order_id = None
+            # else:
+            #    logger.warning(
+            #        f'Got different open_order_id {self.open_order_id} != {order.order_id}')
+            # TODO validate if this is still relevant
             # condition to avoid reset value when updating fees
-            if self.open_order_id == order.order_id:
-                self.open_order_id = None
-            else:
-                logger.warning(
-                    f'Got different open_order_id {self.open_order_id} != {order.order_id}')
+            # if self.open_order_id == order.order_id:
+            #    self.open_order_id = None
+            # else:
+            #    logger.warning(
+            #        f'Got different open_order_id {self.open_order_id} != {order.order_id}')
             self.recalc_trade_from_orders()
         elif order.ft_order_side == self.exit_side:
             if self.is_open:
                 payment = "BUY" if self.is_short else "SELL"
                 # * On margin shorts, you buy a little bit more than the amount (amount + interest)
                 logger.info(f'{order.order_type.upper()}_{payment} has been fulfilled for {self}.')
             # condition to avoid reset value when updating fees
-            if self.open_order_id == order.order_id:
-                self.open_order_id = None
-            else:
-                logger.warning(
-                    f'Got different open_order_id {self.open_order_id} != {order.order_id}')
+            # if self.open_order_id == order.order_id:
+            #    self.open_order_id = None
+            # else:
+            #    logger.warning(
+            #        f'Got different open_order_id {self.open_order_id} != {order.order_id}')
```

Comment:
you can't perform these tests now anymore - so i don't think they're relevant any longer.